SurveyTests/Sources/Specs/Data/Repositories/CachedRepositorySpec.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  CachedRepositorySpec.swift
+//  Survey
+//
+//  Created by Khanh on 08/12/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+// swiftlint:disable all
+
+import Combine
+import Nimble
+import Quick
+import SwiftUI
+
+@testable import Survey
+
+final class CachedRepositorySpec: QuickSpec {
+
+    override func spec() {
+        var repository: CachedRepository!
+        var storage: SurveyListStorageMock!
+
+        describe("a CachedRepository") {
+
+            beforeEach {
+                storage = SurveyListStorageMock()
+                repository = CachedRepository(surveyListStorage: storage)
+            }
+
+            describe("its getData") {
+
+                let expectedData = APISurvey.dummyList
+
+                beforeEach {
+                    storage.performGetDataReturnValue = APISurvey.dummyList
+                    _ = repository.getData()
+                }
+
+                it("gets called") {
```

Comment:
Please be more descriptive of what is called
```suggestion
                it("storage calls GetData") {
```